File: exercises/beecrowd/1116/1116.go
// https://judge.beecrowd.com/pt/problems/view/1116
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)

	for i := 0; i < n; i++ {
		var x, y int
		fmt.Scan(&x, &y)

		if y == 0 {
			fmt.Println("divisao impossivel")
		} else {
			fmt.Printf("%.1f\n", float64(x)/float64(y))
		}
	}
}

File: exercises/beecrowd/1116/1116.java
// https://judge.beecrowd.com/pt/problems/view/1116
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int tests = sc.nextInt();

        for (int i = 0; i < tests; i++) {
            double x = sc.nextDouble();
            double y = sc.nextDouble();

            if (y == 0.0) {
                System.out.println("divisao impossivel");
            } else {
                System.out.printf("%.1f%n", x / y);
            }
        }

        sc.close();
    }
}
